app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -40,6 +52,21 @@
             </li>
           <% end %>
         </ol>
+        <% if rest_of_history.any? %>
+          <div data-module="toggle">
+            <a href="#change-notes-rest-of-history" data-expanded="false" data-controls="change-notes-rest-of-history">
+              Show full history
+            </a>
+            <ol id="change-notes-rest-of-history" class="js-hidden">
```

Comment:
Similar to the comment above, it would be good for this to be an optional parameter